nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/Media.java

```diff
@@ -0,0 +1,53 @@
+package no.unit.nva.model.contexttypes;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import nva.commons.core.JacocoGenerated;
+
+import java.util.Objects;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class Media implements PublicationContext {
+    public static final String PROGRAMME = "programme";
+    public static final String PLACE = "place";
+
+    @JsonProperty(PLACE)
+    private final String place;
+    @JsonProperty(PROGRAMME)
```

Comment:
The  "programme" here is in a way connected to the associated publication instance.